makefile.shade

```diff
@@ -40,4 +40,4 @@ use-standard-lifecycle
    }
 
   var args = '--drop-folder ${BUILD_SHARE} --build-branch ${BUILD_BRANCH} --output-path ${BUILD_DIR} --symbols-source-path ${NUGET_SYMBOLS_SHARE} --symbols-output-path ${SYMBOLS_DIR} --symbols-nuget-exe ${SYMBOLS_NUGET_EXE} --nuget-publish-feed ${NUGET_PUBLISH_FEED} --apikey ${APIKEY} --ci-volatile-share ${PACKAGE_DROP_SHARE}'
-  exec program='cmd' commandline='/C dnx CoherenceBuild ${args}' workingdir='${COHERENCE_BUILD_PROJ}'
+  exec program='dotnet' commandline='run -f dnx451 -- ${args}' workingdir='${COHERENCE_BUILD_PROJ}'
```

Comment:
I thought it would, but no - it complains about not finding a DNX Core target to run.